src/lib/y2firewall/widgets/overview.rb

```diff
@@ -75,7 +75,7 @@ def items
 
       # @return [CWM::PagerTreeItem]
       def startup_item
-        page = Pages::Startup.new(self)
+        page = Pages::Startup.new
```

Comment:
I am fine with removing the `_pager` argument. But we should be consistent and do the same for the rest of pages in this module.